addons/website/views/website_views.xml

```diff
@@ -317,13 +317,15 @@
             <field name="model_id" ref="website.model_website"/>
             <field name="state">code</field>
             <field name="code">action = model.action_dashboard_redirect()</field>
+            <field name="groups_id" eval="[(4, ref('base.group_user'))]"/>
         </record>
 
         <record id="ir_actions_server_website_google_analytics" model="ir.actions.server">
-            <field name="name">Website: Dashboard</field>
+            <field name="name">Website: Analytics</field>
```

Comment:
This really useful?

Reply:
I got the error during debugging and was not able to find which action was problematic, but not really the subject of the PR, true